target_x: 2.8
target_y: 0.0

target1_x: 5.0
target1_y: 0.0

err_yaw: 1